### PR TITLE
test rotating certs

### PR DESCRIPTION
**Pull request by MiaAltieri**
## Issue
No test for rotating TLS certs

## Solution
Add one

### CODE REVIEW

File: tests/integration/tls_tests/helpers.py [hunk 1]
```diff
@@ -81,3 +91,160 @@ async def check_tls(ops_test, unit, enabled) -> None:
                 return True
     except RetryError:
         return False
+
+
+async def time_file_created(ops_test: OpsTest, unit_name: str, path: str) -> int:
```
Comment (MiaAltieri) [inline tests/integration/tls_tests/helpers.py:96]: all these helpers copied from MongoDB charm

**Review by phvalguima — DISMISSED** [review]
LGTM

File: tests/integration/tls_tests/test_tls.py [hunk 2]
```diff
@@ -234,3 +240,64 @@ async def integrate_cluster_with_tls(ops_test: OpsTest) -> None:
...
+
+        check_certs_correctly_distributed(ops_test, unit, app_name=MONGOS_APP_NAME)
+
+        assert (
+            new_external_cert_time > original_tls_times[unit.name]["external_cert"]
+        ), f"external cert for {unit.name} was not updated."
+        assert (
+            new_internal_cert_time > original_tls_times[unit.name]["internal_cert"]
+        ), f"internal cert for {unit.name} was not updated."
+
```
Comment (Mehdi-Bendriss) [inline tests/integration/tls_tests/test_tls.py:312]: Could we please add a check the new cert is different than the old one?

**Review by Mehdi-Bendriss — COMMENTED** [review]
Thanks Mia! I have one comment

**Review by Mehdi-Bendriss — APPROVED** [review]
Thank you!

**Review by phvalguima — APPROVED** [review]

**Pull request merged**